### DISCUSSION TITLE
updateListing returns 404 with message "resource not found" while update state

### DISCUSSION THREAD

**Discussion by bytestechnolab** [etsy/open-api#921] [Q&A]
We do have an application which helps customer to update inventory and listing. While migrating to V3 endpoint for updateListing, we always receive 404 error as "resource not found". Though we are able to get listing details in getListingInventory request.

As per documentation, request we are making as below:
1. https://api.etsy.com/v3/application/shops/xxxxx/listings/yyyy
2. "Content-Type: application/x-www-form-urlencoded"
3. "PATCH" as method
4. parameters "state=active"

Response we receive is:
HTTP/1.1 404 Not Found
...
{"error": "Resource not found"}

**Comment by emergingdzns** [comment]
@bytestechnolab Are you sure you're including the oauth 2 token for the shop?

**Reply by bytestechnolab** [reply]
Yes,request is with oAuth2 tokens for the shop. The reason I am sure is because prior to "PATCH" request, application makes request to get listing inventory by below endpoint:
https://api.etsy.com/v3/application/listings/yyyy/inventory?includes=Listing

**Reply by bytestechnolab** [reply]
Btw, are you able to get correct response code on endpoint? Since 404 response code is believed to be endpoint incorrect.

**Comment by bytestechnolab** [comment]
@emergingdzns is there any other issue with endpoint?

**Reply by emergingdzns** [reply]
As far as I can tell the endpoint works just fine. I can't really tell what's happening with yours.

**Reply by bytestechnolab** [reply]
Can you please share your request params? Not sure if I am making any parameter error? 
Thanks in advance

**Comment by bytestechnolab** [comment]
Is it possible that endpoint responds back as 404 if the listing is already in inactive state?

**Comment by emergingdzns** [comment]
Can you post your full curl command? Obviously take out your API key and token. But if you can provide the rest of the curl function, I can maybe help.
